app/Resources/views/macros/wiki.html.twig

```diff
@@ -217,17 +217,17 @@
     {{ extLink(url, msg('pageviews'), label) }}
 {% endspaceless %}{% endmacro %}
 
-{% macro pageviewsLinks(page) %}{% spaceless %}
+{% macro pageviewsLinks(page, start, end) %}{% spaceless %}
```

Comment:
You might consider setting the defaults `start = null, end = null`, then we don't need the null checks at https://github.com/x-tools/xtools/pull/149/files#diff-fefe304fbb1893557366c352e22673cfR42